Face/Verify/Program.cs

```diff
@@ -0,0 +1,289 @@
+                    : $"Faces from {SourceImageFileName1} & {TargetImageFileNames[0]} are of different (Negative) persons, similarity confidence: {verifyResult1.Confidence}.");
+
+            // Verification example for faces of different persons.
+            VerifyResult verifyResult2 = await Client.Face.VerifyFaceToFaceAsync(sourceFaceId2, targetFaceIds[0]);
+            Console.WriteLine(
+                verifyResult2.IsIdentical
+                    ? $"Faces from {SourceImageFileName2} & {TargetImageFileNames[0]} are of the same (Negative) person, similarity confidence: {verifyResult2.Confidence}."
+                    : $"Faces from {SourceImageFileName2} & {TargetImageFileNames[0]} are of different (Positive) persons, similarity confidence: {verifyResult2.Confidence}.");
+        }
+
+        public static async Task Verify_in_PersonGroup()
+        {
+            // Create a person group.
+            string GroupId = Guid.NewGuid().ToString();
+            Console.WriteLine($"Create a person group ({GroupId}) {Environment.NewLine}");
+            await Client.PersonGroup.CreateAsync(GroupId, GroupId);
+
+            // Create a person group person.
+            Person p = new Person();
+            p.Name = "Dad";
+            p.UserData = "Person for sample";
+            Console.WriteLine($"Create a person group person '{p.Name}' {Environment.NewLine}");
+            p.PersonId = (await Client.PersonGroupPerson.CreateAsync(GroupId, p.Name)).PersonId;
+
+            foreach (var targetImageFileName in TargetImageFileNames)
+            {
+                // Read image file. 
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open))
+                {
+                    // Add face to the person group. 
+                    Console.WriteLine($"Add faces to the person group from image `{targetImageFileName}`.");
+                    PersistedFace faces = await Client.PersonGroupPerson.AddFaceFromStreamAsync(GroupId, p.PersonId, stream, targetImageFileName);
+
+                    if (faces == null)
+                    {
+                        Console.WriteLine($"[Error] No persisted face from image `{targetImageFileName}`.");
+                        return;
+                    }
+                }
+            }
+
+            // Start to train the person group.
+            await Client.PersonGroup.TrainAsync(GroupId);
+
+            // Wait until the training is completed.
+            while (true)
+            {
+                await Task.Delay(1000);
+                var status = await Client.PersonGroup.GetTrainingStatusAsync(GroupId);
+                Console.WriteLine($"Response: Success. Group \"{GroupId}' training process is {status.Status}");
+                if (status.Status != TrainingStatusType.Running)
+                {
+                    break;
+                }
+            }
```

Comment:
No need to train before verification. Both for PersonGroup and LargePersonGroup.

Reply:
done